src/app/components/speakers/speakers.js

```diff
@@ -1,47 +1,43 @@
-
 import React from 'react';
 import { speakersData } from "assets/data/speakers-info.js";
 import './speakers.scss'
 
-const Speaker = (name, title, photo, index) => {
-    return (
-        <div className="speaker-img-wrapper">
-            <img 
-                src = {require('assets/images/speaker-photos/'+photo+'.jpeg')}
-                alt = {name + "'s profile picture"}
-            />
-            <div className="text">
-                <h4>{name}</h4>
-                <p>{title}</p>
-            </div>
-        </div>
-    );
-}
+export default function Speakers() {
 
-const renderSpeakers = (speakers) => {
-    return (
-        <div className="speaker-photos">
-            {speakers.map((speaker, index) => 
-                Speaker(speaker.name, speaker.title, speaker.photo, index)
-            )}
-        </div>
-    ) 
-}
+    function Speaker(name, title, photo, index) {
+        return (
+            <div className="speaker-img-wrapper">
+                <img 
+                    src = {require('assets/images/speaker-photos/'+photo+'.jpeg')}
+                    alt = {name + "'s profile picture"}
+                />
+                <div className="text">
+                    <h4>{name}</h4>
+                    <p>{title}</p>
+                </div>
+            </div>
+        )
+    }
 
-export default class Speakers extends React.Component {
-    render() {
+    function renderSpeakers(speakers) {
         return (
-            <div id="speakers">
-                <p className="subheader">Opening Speakers</p>
-                {renderSpeakers(speakersData.opening)}
-                <p className="subheader">Closing Speakers</p>
-                {renderSpeakers(speakersData.closing)}
-                <p className="subheader">Panelists</p>
-                {renderSpeakers(speakersData.panelists)}
+            <div className="speaker-photos">
+                {speakers.map((speaker, index) => 
+                    Speaker(speaker.name, speaker.title, speaker.photo, index)
+                )}
             </div>
         )
     }
-}
 
-// <p className="subheader">Keynote</p>
-// {renderSpeakers(speakersData.keynotes)}
+    return (
+        <div id="speakers">
+            <p className="subheader">Opening Speakers</p>
+            {renderSpeakers(speakersData.opening)}
+            <p className="subheader">Closing Speakers</p>
+            {renderSpeakers(speakersData.closing)}
+            <p className="subheader">Panelists</p>
+            {renderSpeakers(speakersData.panelists)}
+        </div>
+    )
```

Comment:
(small thing) be sure to add semicolons after statements like returns, variable declarations, etc.; it's neater with them in 😅

Reply:
Thanks for all the suggestions! I just pushed the changes so let me know if anything else needs to be changed.